app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -138,120 +124,43 @@ function ReviewIssuesStep({
-      );
-    }
-    if (inPersonCtaVariantActive === 'in_person_variant_b') {
-      return (
-        <Warning
-          heading={t('errors.doc_auth.throttled_heading')}
-          actionText={t('idv.failure.button.warning_variant')}
-          actionOnClick={onWarningPageDismissed}
-          altActionText={t('in_person_proofing.body.cta.button_variant')}
-          altActionOnClick={onInPersonSelected}
-          altHref="#prepare"
-          location="doc_auth_review_issues"
-          remainingAttempts={remainingAttempts}
-          troubleshootingOptions={
-            <DocumentCaptureTroubleshootingOptions
-              location="post_submission_warning"
-              showAlternativeProofingOptions={false}
-              heading={t('components.troubleshooting_options.ipp_heading')}
-            />
-          }
-        >
-          {!!unknownFieldErrors &&
-            unknownFieldErrors
-              .filter((error) => !['front', 'back'].includes(error.field!))
-              .map(({ error }) => <p key={error.message}>{error.message}</p>)}
-
-          {remainingAttempts <= DISPLAY_ATTEMPTS && (
-            <p>
-              {remainingAttempts === 1
-                ? formatWithStrongNoWrap(t('idv.failure.attempts.one_variant_b_html'))
-                : formatWithStrongNoWrap(
-                    t('idv.failure.attempts.other_variant_b_html', { count: remainingAttempts }),
-                  )}
-            </p>
-          )}
-          <p>{t('in_person_proofing.body.cta.prompt_detail_b')}</p>
-        </Warning>
-      );
-    }
-    if (inPersonCtaVariantActive === 'in_person_variant_c') {
-      return (
-        <Warning
-          heading={t('errors.doc_auth.throttled_heading')}
-          actionText={t('idv.failure.button.warning')}
-          actionOnClick={onWarningPageDismissed}
-          location="doc_auth_review_issues"
-          remainingAttempts={remainingAttempts}
-          troubleshootingOptions={
-            <DocumentCaptureTroubleshootingOptions
-              location="post_submission_warning"
-              showAlternativeProofingOptions={false}
-              heading={t('components.troubleshooting_options.ipp_heading')}
-            />
-          }
-        >
-          {!!unknownFieldErrors &&
-            unknownFieldErrors
-              .filter((error) => !['front', 'back'].includes(error.field!))
-              .map(({ error }) => <p key={error.message}>{error.message}</p>)}
 
-          {remainingAttempts <= DISPLAY_ATTEMPTS && (
-            <p>
-              <strong>
-                {remainingAttempts === 1
-                  ? t('idv.failure.attempts.one')
-                  : t('idv.failure.attempts.other', { count: remainingAttempts })}
-              </strong>
-            </p>
-          )}
-        </Warning>
-      );
-    }
+    return (
+      <Warning
+        heading={t('errors.doc_auth.throttled_heading')}
+        actionText={t('idv.failure.button.warning_variant')}
```

Comment:
Should this key still be called a "variant", or should it be the default?

I think we can't just remove the suffix, since the "Try again" plain text key is used for other things, but maybe we'd want to have a totally new key for the default IPP "Try again online" text.

```suggestion
        actionText={t('iin_person_proofing.cta.button.try_online)}
```